libs/langchain-community/src/vectorstores/cassandra.ts

```diff
@@ -649,4 +730,113 @@ export class CassandraStore extends VectorStore {
     // Wait for all insert operations to complete.
```

Comment:
@jacoblee93 I was not entirely sure how to best handle this tbh. Rather than requiring a user to provide the secure bundle (a .zip file with TLS and other connection-related metadata), this can be downloaded via a service call. I noticed some other uses of axios but none in community. I'm caching this in a tmp subdirectory.

Reply:
I'm not sure but we should use the native `fetch` object instead of Axios if possible

Reply:
Ah, sorry missed that this is referencing the below code